tensorflow/core/kernels/data/experimental/snapshot_util.cc

```diff
@@ -904,9 +906,10 @@ Status DetermineOpState(const std::string& mode_string, bool file_exists,
     return Status::OK();
   }
 
-  if (metadata->creation_timestamp() >=
-      (static_cast<int64>(EnvTime::NowMicros()) -
-       pending_snapshot_expiry_seconds * 1000000)) {
+  int64 expiration_timer = EnvTime::NowMicros() 
+  		     - pending_snapshot_expiry_seconds * 1000000;
```

Comment:
We still need the existing cast since [`EnvTime::NowMicros` returns `uint64`](https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/core/platform/env.h;l=289;drc=fedc6d951faa73936a1154d6507d54240614d416)

Reply:
updated.